modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -972,7 +972,7 @@ module.exports = (self) => {
     // properties is handled at a lower level in a beforeSave
     // handler of the doc-type module.
 
-    async convert(req, field, data, destination) {
+    async convert(req, field, data, destination, options = { fetchRelationships: true }) {
```

Comment:
As we were talking about this morning, use the `{ fetchRelationships: true } = {}` syntax to gracefully handle both cases where no object is passed at all and cases where the object is passed but this particular option has no specified value.

Reply:
Yes I did but options is reused below. I'll create a constant for that.

Reply:
Done